src/pages/Pricing/Questions/index.tsx

```diff
@@ -1,33 +1,33 @@
 import { Accordion, Container, Text, Title } from "@mantine/core";
 
-export default function Questions() {
-  const faqs = [
-    {
-      value: "q1",
-      question: "Can I cancel my subscription?",
-      answer:
-        "Yes, you can cancel your subscription at any time. Once you cancel, you’ll be downgraded to the free plan at the end of your billing cycle.",
-    },
-    {
-      value: "q2",
-      question: "Do you offer discounts for students?",
-      answer:
-        "Yes, we offer a 50% discount on all plans for students. Simply sign up with your student email and the discount will be applied automatically.",
-    },
-    {
-      value: "q3",
-      question: "Can I change my plan later on?",
-      answer:
-        "Yes, you can change your plan at any time. Simply go to your account settings and select a new plan.",
-    },
-    {
-      value: "q4",
-      question: "Do you offer refunds?",
-      answer:
-        "We do not offer refunds. If you cancel your subscription, you’ll be able to use vSuS until the end of your billing cycle.",
-    },
-  ];
+const faqs: { value: string; question: string; answer: string }[] = [
```

Comment:
what is `value`?

Reply:
basically to help manage accordion state
[read more](https://mantine.dev/core/accordion)